src/loader.ts

```diff
@@ -292,6 +294,48 @@ declare const Packages: {} | undefined;
 
 			// Note that old paths will get destroyed if reconfigured
 			configuration.paths && (pathMapPrograms = computeMapProgram(configuration.paths));
+
+			// shim API
+			if (config.shim !== undefined) {
+				Object.keys(config.shim).forEach((moduleId) => {
+					let moduleDef: ModuleShim = (config.shim || {})[ moduleId ];
+
+					// using shorthand module syntax, convert to full syntax
+					if ('length' in moduleDef) {
```

Comment:
assuming you're checking to see if this is an array, would `Array.isArray` not be better?